src/@next/components/organisms/StripePaymentGateway/StripePaymentGateway.tsx

```diff
@@ -1,24 +1,64 @@
 import { CardNumberElement, Elements } from "@stripe/react-stripe-js";
-import { loadStripe, Stripe, StripeElements } from "@stripe/stripe-js";
-import React, { useMemo, useState } from "react";
+import {
+  loadStripe,
+  PaymentMethod,
+  Stripe,
+  StripeElements,
+} from "@stripe/stripe-js";
+import React, { useEffect, useMemo, useState } from "react";
+import { defineMessages, useIntl } from "react-intl";
 
+import { paymentErrorMessages } from "@temp/intl";
 import { IFormError } from "@types";
 
 import { StripeCreditCardForm } from "../StripeCreditCardForm";
 import { IProps } from "./types";
 
+const messageDescription = "Stripe payment gateway error";
+
+export const stripeErrorMessages = defineMessages({
```

Comment:
Maybe this stuff could go into intl.ts?

Reply:
Sure, moved to separate intl.ts file for Stripe